menstruation/config.py

```diff
@@ -146,4 +95,14 @@ def remove_user(self, user_id: int) -> int:
         return self.redis.hdel(str(user_id), 'mensa', 'subscribed', 'menu_filter')
 
 
-menstruation_config = MenstruationConfig()
+args = get_environment_args()
```

Comment:
do we have to construct a dictionary just to deconstruct it later?
why not just get `token` etc straight from the environment?